src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/PooledArrayBufferWriter`1.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+// Copied from https://github.com/dotnet/runtime
```

Comment:
Can you please provide the exact link, including sha?

Reply:
Sure thing!